tests/lang/s01_basics/s00_atoms/t03_compiletime.nim

```diff
@@ -0,0 +1,50 @@
+# Given this is a specification I don't think it is an appropriate place to
```

Comment:
```suggestion
discard """
output: '''
echo outside of static
'''
"""

# Given this is a specification I don't think it is an appropriate place to
```

This one needs an output specifier or megatest will complain about extra output.